clients/remote/server/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "server",
+  "version": "1.0.0",
+  "description": "",
+  "main": "src/index.ts",
+  "scripts": {
+    "build": "tsc",
+    "dev": "ts-node src/index.ts",
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "node dist/index.js"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "type": "commonjs",
```

Comment:
we should try to aim to make everything work with ES Modules (the new ecosystem standard) rather than CommonJS. I think if you change this to `"type": "module"` (and in the tsconfig), you should hopefully be able to build and run it successfully